content/en/tracing/setup/php.md

```diff
@@ -112,13 +114,17 @@ SetEnv DD_TRACE_DEBUG true
 
 ### NGINX
 
-For NGINX, use the `env` directory in the php-fpm's `www.conf` file, for example:
+For NGINX, use the `env` directive in the php-fpm's `www.conf` file, for example:
 
 ```
-env[DD_AGENT_HOST] = $FROM_HOST_ENV
-env[DD_TRACE_DEBUG] = true
+; Both actual values (e.g. localhost) and references
+; to env variables (e.g. $SOME_ENV) can be used here
+env[DD_AGENT_HOST] = $SOME_ENV
+env[DD_SERVICE] = my-app
 ```
 
+**Note**: If you have enabled APM for your NGINX server, make sure you have properly configured the `opentracing_propagate_context` setting for distributed tracing to properly work. See [NGINX APM configuration][14] for more details.
```

Comment:
I think it'd be clearer to directly mention the `opentracing_fastcgi_propagate_context` directive here instead, because `opentracing_propagate_context` is the one that behaves badly.
Maybe this bit in nginx docs should be its own heading named `PHP-FPM / FastCGI`, and the link to that specific heading?
> When the location is serving a FastCGI backend instead of HTTP, the location block should use opentracing_fastcgi_propagate_context instead of opentracing_propagate_context.

Reply:
Good catch and good idea. let me change this.

Reply:
The change I proposed adds a header the the nginx page (so I can link from the php page) and links to it. @cgilmour what do you think?

Reply:
Yup, it's exactly what I was thinking 😄